Lib/test/support/__init__.py

```diff
@@ -2070,21 +2072,48 @@ def _is_full_match_test(pattern):
     return ('.' in pattern) and (not re.search(r'[?*\[\]]', pattern))
 
 
-def set_match_tests(patterns):
-    global _match_test_func, _match_test_patterns
+def set_match_tests(accept_patterns, reject_patterns):
```

Comment:
Maybe use None as default value to "document" that None is accepted:

```suggestion
 def set_match_tests(accept_patterns=None, reject_patterns=None):
```